Tests/W3CTestSuite/svg/__issue-785-01.svg

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<svg viewBox="0 0 200 200" xmlns="http://www.w3.org/2000/svg">
+  <polyline style="fill: red; stroke:black; stroke-width: 5px;" points="10 20 35 35 60 20 35 10 10 20"/>
+  <polyline style="fill: red; stroke:black; stroke-width: 0px;" points="110 20 135 35 160 20 135 10 110 20"/>
+  <circle style="fill: blue; stroke:black; stroke-width: 5px;" cx="35" cy="70" r="20"/>
+  <circle style="fill: blue; stroke:black; stroke-width: 0px;" cx="135" cy="70" r="20"/>
+<rect style="fill: green; stroke:black; stroke-width: 5px;" x="20" y="105" width="30" height="30"/>
```

Comment:
Please insert indent \<rect\> as well.

Reply:
I actually had tested it, but forgot to add it to the file - good point.